.rubocop.yml

```diff
@@ -23,6 +23,7 @@ Layout/LineLength:
     - db/seeds.rb
     - spec/factories/users.rb
     - spec/requests/api/v1/**/*
+    - spec/models//**/*
```

Comment:
does this work?
shouldn't it be sth like `- spec/models/**/*`?

Reply:
It works. It seems that ignore the extra slash. BTW, now its fixed.